adafruit_tlc5947.py

```diff
@@ -50,6 +50,8 @@
 # access is by design for the internal class.
 #pylint: disable=protected-access
 
+_CHANNELS = 24
+_STOREBYTES = _CHANNELS + _CHANNELS/2
```

Comment:
```suggestion
_STOREBYTES = int(_CHANNELS + _CHANNELS/2)
```

Reply:
...otherwise we wind up with a float here, at least on CPython.

Reply:
Thank you for finding this one! I hoped to avoid this pitfall by rephrasing `_STOREBYTES = _CHANNELS * 1.5` but forgot about [Python 3 division rules](https://stackoverflow.com/questions/1282945/python-integer-division-yields-float). So here is my fix:
```suggestion
_STOREBYTES = _CHANNELS + _CHANNELS//2
```
which I find less bulky, but also a bit less clear.